### ISSUE TITLE
Problems running nvitop on Windows after installation

### ISSUE THREAD

**Issue by WMRamadan** [XuehaiPan/nvitop#2]
#### Runtime Environment

- Operating system and version: Windows 11 Pro version 21H2 (OS Build 22000.65)
- Terminal emulator and version: Windows PowerShell 5.1.22000.65
- Python version: 3.9.6
- NVML/CUDA version: NVIDIA-SMI 471.21       Driver Version: 471.21       CUDA Version: 11.4
- `nvitop` version/commit: 0.3.5.5
- `python-ml-py` version: 11.450.51
- Locale: ENG

#### Current Behavior

1 - After installing nvitop as instructed you cannot launch nvitop by running "nvitop" command in Windows Command Prompt or Windows PowerShell. Current solution is to use "python -m nvitop" command to run.

2 - Running nvitop using "python -m nvitop" produces error "ModuleNotFoundError: No module named '_curses'". Solution is to install windows-curses using "pip3 install windows-curses"

#### Expected Behavior

1 - nvitop command should be executable directly from Windows Command Prompt or Windows PowerShell after installation.

2 - pip should install all necessary packages needed for Windows such as "windows-curses" package.

#### Context

Unless you are experienced with Python this could be difficult for an average user.

#### Possible Solutions

1 - Use "python -m nvitop" to run nvitop or add to PATH

2 - Install "windows-curses" pip package after installation of nvitop

#### Steps to reproduce

1. Install nvitop on Windows and try to run using "nvitop" command.
2. After installation run nvitop with "python -m nvitop" command.

```

```

**Comment by XuehaiPan** [comment]
Thanks for the feedback!

For the first issue:

> 1. nvitop command should be executable directly from Windows Command Prompt or Windows PowerShell after installation.

The Python executable (in `PYTHONHOMEDIR`) and the scripts installed by `pip` (in `PYTHONHOMEDIR/Scripts`) are in the different directory, You should add both Python home and its script directory to your shell PATH. (If you are using conda or virtualenv, these entries will be added to your PATH at activation).

<img width="1440" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/16078332/125660051-e5971724-3d97-4c8d-bd16-cfbabe93bbaa.png">

For the second issue:

> 2. pip should install all necessary packages needed for Windows such as "windows-curses" package.

I tested and packed `nvitop` on my Linux machine, and the `sys.platform` was `Linux` when I build the wheel. When you installing `nvitop` on Windows from PyPI, `pip` will use the wheel file instead of running the `setup.py` which will determine `sys.platform` at runtime. I will repack the wheel file to PyPI later.

<img width="1440" alt="Screen Shot 2" src="https://user-images.githubusercontent.com/16078332/125660090-dcb0afd4-8710-4eb1-b8a9-1776e21978d7.png">

**Comment by XuehaiPan** [comment]
I have uploaded the repacked version of `nvitop` (fixed by b669fa39b161e7d482221ccee696d8dd82ed8910) to the PyPI index, you can try:

```bash
pip3 uninstall windows-curses
pip3 install nvitop==0.3.5.6
```

It should install `windows-curses` automatically.

**Comment by XuehaiPan** [comment]
Thanks for the feedback from @WMRamadan! Please feel free to reopen this issue if you have any other question.
